osmosis-types/gamm/genesis.pb.go

```diff
@@ -9,8 +9,8 @@ import (
 	types1 "github.com/cosmos/cosmos-sdk/codec/types"
 	github_com_cosmos_cosmos_sdk_types "github.com/cosmos/cosmos-sdk/types"
 	types "github.com/cosmos/cosmos-sdk/types"
-	_ "github.com/gogo/protobuf/gogoproto"
-	proto "github.com/gogo/protobuf/proto"
+	_ "github.com/cosmos/gogoproto/gogoproto"
```

Comment:
Hey @aljo242, how did you update this? By generating code or by hand ser?

Reply:
Yes, by hand to first remove the `gogo/protobuf` import entirely